catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/util/impl/QueryResultIterable.java

```diff
@@ -131,9 +136,17 @@ private void queryNext(int index) {
                 Map<String, Serializable> props = new HashMap<>();
```

Comment:
Found it belongs in QueryRequestImpl but we were using an overloaded method which didn't require it.